docs/source/Plugins.rst

```diff
@@ -157,6 +157,20 @@ plugin (``json``) will be executed before the XUnit plugin (``xunit``).
 If the HTML result plugin is installed and enabled (``html``) it will
 be executed before both JSON and XUnit.
 
+Configuring the plugin execution order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On some circumstances it may be necessary to change the order in which plugins
+are executed.  To do so, add a ``order`` entry a configuration file section
+named after the plugin type.  For ``job.prepost`` plugin types, the section name
+has to be named ``plugins.job.prepost``, and it would look like this::
+
+  [plugins.job.prepost]
+  order = ['myplugin', 'jobscripts']
```

Comment:
For the first example (or use case) it's way off the scope we intend to deliver here.  The fact is that the points at which those actions are called are hard coded (think of them as pre defined hooks).  To achieve what we suggest, one approach would be to normalize all plugin types, so that they are much more generic, receive the same parameters and could be called at every single hook.  The plugin code would have to understand where it's being called from and interfaces would not mean much.  I must say I don't like that approach.

Wrt to the second use case, you can *disable* any plugin (any type) with the `disable = ` key introduced in the previous releases.